src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -13,28 +14,29 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>Serilog.Settings.Configuration</PackageId>
     <PackageTags>serilog;json</PackageTags>
-    <PackageIconUrl>https://serilog.net/images/serilog-configuration-nuget.png</PackageIconUrl>
+    <PackageIcon>icon.png</PackageIcon>
     <PackageProjectUrl>https://github.com/serilog/serilog-settings-configuration</PackageProjectUrl>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/serilog/serilog-settings-configuration</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <RootNamespace>Serilog</RootNamespace>
   </PropertyGroup>
 
-  <PropertyGroup Condition="('$(TargetFramework)' == 'net451') Or ('$(TargetFramework)' == 'net461')">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net451' or '$(TargetFramework)' == 'net452' or '$(TargetFramework)' == 'net461'">
     <DefineConstants>$(DefineConstants);PRIVATE_BIN</DefineConstants>
```

Comment:
Is there any specific reason for a new tfm? AFAIR, the only reason for `net451` target is because `Microsoft.Extensions.DependencyMode` has it (otherwise it would be `net45`)

Reply:
I think that it already was duscussed in #201

Reply:
I see. I think `net452` was added to test project for sake of `xunit.execution.desktop` package. Otherwise, it won't discover tests under Windows for full .net tfm.  So its not related to this package

Reply:
I no longer remember exactly the sequence of my actions. I remember that there was some kind of problem that did not allow me to build solution. I began to deal with conditional multitargeting and saw that it differs between projects in the solution - somewhere was 451, somewhere 452/461. As a result, I tried to unify it. That's all.

Reply:
@skomis-mm I reverted my changes on TFMs.